sdk/src/beta9/runner/common.py

```diff
@@ -210,18 +211,26 @@ def end_task_and_send_callback(
         ),
         payload=payload,
         task_status=end_task_request.task_status,
+        override_callback_url=override_callback_url,
     )
 
     return resp
 
 
 def send_callback(
-    *, gateway_stub: GatewayServiceStub, context: FunctionContext, payload: Any, task_status: str
+    *,
+    gateway_stub: GatewayServiceStub,
+    context: FunctionContext,
+    payload: Any,
+    task_status: str,
+    override_callback_url: Optional[str] = None,
 ) -> None:
     """
     Send a signed callback request to an external host defined by the user
     """
-    if context.callback_url == "" or context.callback_url is None:
+
+    callback_url = override_callback_url or context.callback_url
+    if callback_url == "" or callback_url is None:
```

Comment:
Can can we simplify this with `if not callback:`? That should cover these two conditions and any other data type condition that is falsy.

Reply:
yea you are right.